packages/components/organisms/f-takeawaypay-activation/src/components/_tests/TakeawaypayActivation.test.js

```diff
@@ -1,10 +1,34 @@
-import { shallowMount } from '@vue/test-utils';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import { VueI18n } from '@justeat/f-globalisation';
 import TakeawaypayActivation from '../TakeawaypayActivation.vue';
+import tenantConfigs from '../../tenants';
+
+const i18n = {
+    locale: 'en-GB',
+    messages: {
+        'en-GB': tenantConfigs['en-GB'].messages
+    },
+    dateTimeFormats: {
+        'en-GB': tenantConfigs['en-GB'].dateTimeFormats
```

Comment:
I can't see `dateTimeFormats` defined in `en-GB.js`?

Reply:
Yeah, not needed here.